packages/suite/src/components/wallet/CoinjoinSummary/index.tsx

```diff
@@ -24,7 +24,7 @@ export const CoinjoinSummary = ({ account }: CoinjoinSummaryProps) => {
             {session ? (
                 <CoinjoinSessionStatus account={account} session={session} />
             ) : (
-                <CoinjoinSetup account={account} />
+                <AccountSummarySection />
```

Comment:
Would be cool if you get rid of that render condition and lift the whole `SummarySection` up there. Since the `BalanceSection` and `AnonymityChart` are actually visible during an ongoing section too, the session specific component with the status and all should be in the `BalanceSection`, instead of the `AnonymizeButton`. 

For now you can render the `CoinjoinSetup` instead of the whole `BalanceSection` in the `AccountSummarySection`.

Makes sense? 😅

Reply:
I believe i got it :) fcbc1400acbd07df4d7541c35a689a99b9babf0d
I also reworked the file structure a bit in e21d1f032f3760ca7d4a76b8cf96a14314700bdd